idd/Energy+.idd.in

```diff
@@ -52927,6 +53038,337 @@ AirConditioner:VariableRefrigerantFlow,
+       \units C       
+       \default 4.0
+       \note This field is used if Refrigerant Temperature Control Algorithm 
+       \note is VariableTemp 
+  N8 , \field Variable Evaporating Temperature Maximum for Indoor Unit
+       \type real
+       \units C       
+       \default 13.0
+       \note This field is used if Refrigerant Temperature Control Algorithm 
+       \note is VariableTemp 
+  N9 , \field Variable Condensing Temperature Minimum for Indoor Unit
+       \type real
+       \units C       
+       \default 42.0
+       \note This field is used if Refrigerant Temperature Control Algorithm 
+       \note is VariableTemp
+  N10, \field Variable Condensing Temperature Maximum for Indoor Unit
+       \type real
+       \units C       
+       \default 46.0
+       \note This field is used if Refrigerant Temperature Control Algorithm 
+       \note is VariableTemp
+  A6 , \field Outdoor Unit Fan Object Type
+       \type choice 
+       \key Fan:OnOff
+       \key Fan:ConstantVolume 
+       \default Fan:ConstantVolume 
+  A7 , \field Outdoor Unit Fan Object Name
+       \required-field
+       \type object-list
+       \object-list FansCVandOnOff
+  A8 , \field Outdoor Unit Evaporating Temperature Function of Superheating Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves
+       \object-list UniVariateTables
+  A9 , \field Outdoor Unit Condensing Temperature Function of Subcooling Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves	 
+       \object-list UniVariateTables  
+  N11, \field Diameter of main pipe connecting outdoor unit to indoor units
+       \required-field
+       \note used to calculate the piping loss
+       \type real 
+       \units m
+       \minimum 0.0
+  N12, \field Length of main pipe connecting outdoor unit to indoor units
+       \required-field
+       \note used to calculate the heat loss of the main pipe
+       \type real 
+       \units m
+       \minimum 0.0
+  N13, \field Equivalent length of main pipe connecting outdoor unit to indoor units
+       \required-field
+       \note used to calculate the refrigerant pressure drop of the main pipe
+       \type real 
+       \units m
+       \minimum 0.0
+  N14, \field Height difference between the outdoor unit node and 
+       \field indoor unit node of the main pipe
+       \required-field
+       \note Difference between outdoor unit height and indoor unit height
+       \note Positive means outdoor unit is higher than indoor unit 
+       \note Negative means outdoor unit is lower than indoor unit 
+       \type real 
+       \units m
+  N15, \field Insulation thickness of the main pipe
+       \required-field
+       \type real 
+       \units m
+       \minimum 0.0	
+  N16, \field Thermal conductivity of the main pipe insulation material
+       \required-field
+       \type real 
+       \units W/m-K
+       \minimum 0.0	
```

Comment:
It would be better if field N11 - N16 were not required fields and all had a default. I won't force this. For example a 1" pipe diameter and a 30 m pipe length default. Whatever you think for the other inputs. These could be described in the IO Reference for each field.